include/libopencm3/stm32/g4/exti.h

```diff
@@ -0,0 +1,42 @@
+/** @defgroup exti_defines EXTI Defines
+ *
+ * @brief <b>Defined Constants and Types for the STM32F4xx External Interrupts
+ * </b>
+ *
+ * @ingroup STM32F4xx_defines
+ *
+ * @author @htmlonly &copy; @endhtmlonly 2013
+ * Piotr Esden-Tempski <piotr@esden.net>
```

Comment:
no, this is not esden. this is either no copyright, or, if you must, your own name.